### DISCUSSION TITLE
is anybody using Logux for any project or product?

### DISCUSSION THREAD

**Discussion by rozek** [logux/logux#124] [General]
...I was searching for examples but could not find any application of Logux.

Am I blind?

**Comment by ai** [comment]
Yes, I am using it on my current project https://github.com/hplush/slowreader

And also I used it for a few startups: [Akeero](https://evilmartians.com/chronicles/collaborative-real-time-security-logux-for-akeero) and [Amplifr](https://dsokolovskiy.com/blog/all/amplifr/)

**Reply by rozek** [reply]
great - don't you want to mention them in your docs? Other people like me could be interested in these use cases

**Reply by ai** [reply]
Good idea! Also, I am going to add examples link to the docs.

**Reply by ai** [reply]
Done https://logux.org/examples/list/

Thanks for the idea

**Reply by rozek** [reply]
Cool - that's really helpful! Thanks a lot!
